scriptlets/_put

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+#
+# Copyright 2023 Canonical Ltd.
+#
+# This script provides a mechanism for transferring a file to a remote device via SCP.
+# It uses the `set-ssh-options` script to set SSH options and then executes the SCP command.
+# This script is intended to be a drop-in replacement for inline convenience
+# functions previously defined in each of the Jenkins jobs.
```

Comment:
I feel like this should be part of the commit body and not of the script itself.